components/Navbar.tsx

```diff
@@ -48,23 +48,34 @@ export default function Navbar() {
             </a>
           ))}
         </div>
-        <div className="flex flex-1 items-center justify-end gap-x-6">
-          <a
-            href="/#contact"
-            className="rounded-md bg-cyan-600 py-2 px-3 text-sm font-semibold text-white shadow-sm hover:bg-cyan-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-cyan-600"
-          >
-            Contact Me
+        <button className="group flex flex-0  ml-auto items-center justify-end rounded-md text-white bg-cyan-600 py-2 px-3 shadow-sm hover:bg-cyan-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-cyan-600">
+          <a href="/#contact" className="text-sm font-medium">
```

Comment:
**Notice the `group` class on the parent tag?** 
Child tags can use `group-[]:` to change style when something happens to the group. 

**Example:** 
Adding `className="group-hover:text-red-400"` to this <a> will change the text to red when the parent group is hovered. 
The <svg> text color would stay white.